config/bootstrap.php

```diff
@@ -1,3 +1,2 @@
 <?php
-
-\Cake\Routing\DispatcherFactory::add('Muffin/Webservice.ControllerEndpoint');
+\Cake\Datasource\FactoryLocator::add('Endpoint', [Muffin\Webservice\Model\EndpointRegistry::class, 'get']);
```

Comment:
Do we have required integration tests to ensure this change works correctly?

Reply:
Whoops. That's me typing too fast, I've done that in #63 also.

Reply:
With `TableRegistry` being deprecated in core, we should ideally deprecate `EndpointRegistry` too and add and end point locator class similar to table locator :slightly_smiling_face:

Reply:
Good idea, I'll add an item to #51